### ISSUE TITLE
SQL error: ORA-04063: package body "WUN.CITYDB_IDX" 有错误

### ISSUE THREAD

**Issue by iWun** [3dcitydb/3dcitydb#5]
Hi,
I'm new to 3dcitydb.I have set up the database according to the documentation,but when I try to import gml,I met the error:

`ERROR] SQL error: ORA-04063: package body "WUN.CITYDB_IDX" 有错误`
`ORA-06508: PL/SQL: 无法找到正在调用 : "WUN.CITYDB_IDX" 的程序单元`

Can anybody help me? thanks.

**Comment by yaozhihang** [comment]
Hello,
it seems that your 3DCityDB instance has not been set up correctly, that is, the 3DCityDB package "CITYDB_INX" is missing. Have you received any error messages when you ran the CREATE_DB script via the command line interface?

**Comment by clausnagel** [comment]
Closing, since no feedback has been received. Feel free to re-open if your problem persists.

**Comment by jowajowa** [comment]
Hi there,

I have the same problem as Wun. I didn't receive any error massage during compiling. Actually the cmd-window tells me that all the packages have been created without errors.

What can I do?

Greetings, Johannes

**Comment by FxKu** [comment]
I don't know if this is related, but I found out that if you are using a role that has only inherited the CREATE INDEX privilege from another role you are not able to execute the DDL commands within the IDX package. The privilege has to be explicitly assigned to the user.

Another option is to add an AUTHID to the IDX package which I did with [3f9a1b1](https://github.com/3dcitydb/3dcitydb/commit/3f9a1b17644e6c77de4ea668c5137119aeab674c) inside the dev branch.

Simply add AUTHID CURRENT_USER to the package creation command like:
```sql
CREATE OR REPLACE PACKAGE citydb_idx **AUTHID CURRENT_USER**
...
```

Hopefully, this solves your issues. The new code will be merged into the master soon. I would close the issue then in case I do not receive more error reports.
